### ISSUE TITLE
iOS does not support VoiceOver scrub back gesture

### ISSUE THREAD

**Issue by luomo-pro** [flutter/flutter#95567]
In iOS native, VoiceOver can use the gesture of scrub back and forth with two fingers to return to the page.
The steps for scrub gestures are:
1. Press and hold the screen with two fingers, slide to the left first, don't let go.
2. Swipe back with two fingers to the right, and still don't let go.
3. Slide two fingers to the left again, and then let go.

The following is my operation video of returning to the page using the scrub gesture in the iOS native settings:

https://user-images.githubusercontent.com/41097395/146786623-51cc9482-b1d2-4482-a813-f25ffca0798a.mp4

However, in flutter, this gesture cannot be used.
For example, I use the following code to jump to the page:
        Navigator.push( 
          context,
          MaterialPageRoute(builder: (context) {
            return NewRoute();
          }),
        );

In the page after the jump, I tried to return to the page with two fingers, but nothing happened and the page would not return.
I am using the stable version of flutter 2.8.

**Comment by yasargil** [comment]
Duplicate of https://github.com/flutter/flutter/issues/74246

@luomo-pro see here for a workaround 
https://github.com/flutter/flutter/issues/74246#issuecomment-971326926

**Comment by luomo-pro** [comment]
hi
@yasargil
I read this issue, and according to the comments inside, it is possible to solve this problem.
But I think this is not a good plan.
In iOS native, this gesture is supported by default.
It is very important for blind users, and we should not expect everyone from the developer to manually set it up to support it.
Flutter should be supported by default, which will help countless applications that use flutter to support this gesture by default.

**Comment by mamuseferha** [comment]
Hi @luomo-pro, thank you for filing this issue.
Please follow up on this issue https://github.com/flutter/flutter/issues/74246, I'm closing the current one as a duplicate.
If you disagree, please write in the comments and I will reopen it.
Thank you

**Comment by luomo-pro** [comment]
@mamuseferha
Okay, then I will follow up in that issue.
However, I think that the issue was submitted in January this year, and it has been a year since it has not been resolved.
Can you help copy to the person responsible for this? Or raise its priority
After all, this problem has a great impact on blind users, and it is very easy to fix.
It has been a year, and I hope it can be repaired soon.
Many thanks!
